src/tests/functional/plugin/conformance/test_runner/op_conformance_runner/src/op_impl_check/op_impl_check.cpp

```diff
@@ -14,13 +15,21 @@ namespace op {
 using namespace ov::test::subgraph;
 
 namespace {
+INSTANTIATE_TEST_SUITE_P(conformance_compile_model,
+                         OpImplCheckCompileModelTest,
+                         ::testing::Combine(
+                                 ::testing::ValuesIn(createFunctions()),
+                                 ::testing::Values(targetDevice),
+                                 ::testing::Values(pluginConfig)),
+                         OpImplCheckCompileModelTest::getTestCaseName);
+
 INSTANTIATE_TEST_SUITE_P(conformance_query_model,
```

Comment:
I believe, we can remove this test. Because `compile_model` is enough. Moreover, looks like we instansiate the same test suite twice